src/k8s/hack/gen_cmd_docs.go

```diff
@@ -0,0 +1,17 @@
+package main
```

Comment:
this impl of cobra needs to be `go run`, which needs a `main`, which would conflict with our `./src/k8s/cmd/main.go` -- hence this is in our `hack` util dir.